build.gradle.kts

```diff
@@ -13,6 +13,26 @@ plugins {
 group = "com.gradle"
 
 repositories {
+    maven {
+        name = "Solutions"
+        url = uri("https://repo.gradle.org/artifactory/solutions")
+        credentials {
+            username = providers
+                .environmentVariable("GRADLE_SOLUTIONS_REPOSITORY_USERNAME")
+                .orElse(providers.gradleProperty("gradleSolutionsRepositoryUsername"))
+                .get()
+            password = providers
+                .environmentVariable("GRADLE_SOLUTIONS_REPOSITORY_PASSWORD")
+                .orElse(providers.gradleProperty("gradleSolutionsRepositoryPassword"))
+                .get()
+        }
+        authentication {
+            create<BasicAuthentication>("basic")
+        }
```

Comment:
Once the repository is made public, we can drop the credentials and authentication configuration.

Reply:
Does the above code pass when no credentials are specified, or will the build throw an error when get() is called?

Reply:
The build will throw an error when `get()` is called. https://ge.solutions-team.gradle.com/s/sonhpjgtya2lu/failure#1

Reply:
That's what I suspected. Given this is all temporary, it's fine like that.